test/common.py

```diff
@@ -467,20 +477,39 @@ def require_wasm64(self):
       version = shared.check_node_version()
       if version >= (16, 0, 0):
         self.js_engines = [config.NODE_JS]
-        self.node_args += ['--experimental-wasm-memory64']
+        self.node_args.append('--experimental-wasm-memory64')
         return
 
     if config.V8_ENGINE and config.V8_ENGINE in self.js_engines:
       self.emcc_args.append('-sENVIRONMENT=shell')
       self.js_engines = [config.V8_ENGINE]
-      self.v8_args += ['--experimental-wasm-memory64']
+      self.v8_args.append('--experimental-wasm-memory64')
       return
 
     if 'EMTEST_SKIP_WASM64' in os.environ:
       self.skipTest('test requires node >= 16 or d8 (and EMTEST_SKIP_WASM64 is set)')
     else:
       self.fail('either d8 or node >= 16 required to run wasm64 tests.  Use EMTEST_SKIP_WASM64 to skip')
 
+  def require_wasm_eh(self):
+    if config.NODE_JS and config.NODE_JS in self.js_engines:
+      version = shared.check_node_version()
+      if version >= (16, 0, 0):
+        self.js_engines = [config.NODE_JS]
+        if version < (20, 0, 0):
+          self.node_args.append('--experimental-wasm-eh')
+        return
+
+    if config.V8_ENGINE and config.V8_ENGINE in self.js_engines:
+      self.emcc_args.append('-sENVIRONMENT=shell')
+      self.js_engines = [config.V8_ENGINE]
+      self.v8_args.append('--experimental-wasm-eh')
```

Comment:
The latest v8 doesn't seem to accept `--experimental-wasm-eh`; it enables EH by default. I don't know from which version though. I think we need to check the version of v8 as we did for node.

Reply:
It looks like v8 only shows a warning:

```Warning: unknown flag --experimental-wasm-eh.```

I will do a followup to drop support for older versions of v8.